src/org/opensolaris/opengrok/history/FileHistoryCache.java

```diff
@@ -442,7 +455,8 @@ public void store(History history, Repository repository)
                 @Override
                 public void run() {
                     try {
-                        doFileHistory(map_entry, env, repositoryF,
+                        doFileHistory(map_entry.getKey(), map_entry.getValue(),
+                            env, repositoryF,
                             new File(env.getSourceRootPath() + map_entry.getKey()),
                             root, true);
                     } catch (Exception ex) {
```

Comment:
yep, using map_entry to pass the repo name and history made the code hard to read.